.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+<!-- 🚨 Please Do Not skip any instructions and information mentioned below as they are all required and essential to evaluate and test the PR. By fulfilling all the required information you will be able to reduce the volume of questions and most likely help merge the PR faster 🚨 -->
+
+<!-- ⚠ We will not merge the PR to the main repo if your changes are not in a *feature branch* of your forked repository. Create a new branch in your repo, **do not use the `main` branch in your repo**! ⚠ -->
+
+<!-- 👉 It is imperative to resolve ONE ISSUE PER PR and avoid making multiple changes unless the changes interrelate with each other -->
+
+<!-- 📝 Please always keep the "☑️ Allow edits by maintainers" button checked in the Pull Request Template as it increases collaboration with the Toolkit maintainers by permitting commits to your PR branch (only) created from your fork. This can let us quickly make fixes for minor typos or forgotten StyleCop issues during review without needing to wait on you doing extra work. Let us help you help us! 🎉 -->
+
+## Fixes
+
+<!-- Add the relevant issue number after the word "Fixes" mentioned above (for ex: `## Fixes #0000`) which will automatically close the issue once the PR is merged. -->
+
+<!-- Add a brief overview here of the feature/bug & fix. -->
+
+## PR Type
+
+What kind of change does this PR introduce?
+
+<!-- Please uncomment one or more options below that apply to this PR. -->
+
+<!-- - Bugfix -->
+<!-- - Feature -->
```

Comment:
Should add a note about " - Features should come from Windows Community Toolkit Labs, link to the approval comment for migration here" (or maybe have this last bit in the checklist below)

Reply:
Good catch, thought I had all the feature-related things removed.